dags/sustainment/pull_etl_configs.py

```diff
@@ -166,367 +135,354 @@ def get_nifi_configs(**kwargs):
 
-                            airflow_configs.append( {
-                                "source_data" : config[package_name]["resources"][resource_name]["url"],
-                                "target_package_name": package_name,
-                                "target_resource_name": resource_name,
-                                "engine": "Airflow - YAML Job",
-                                "schedule" : cron_to_english( dag.schedule_interval ),
-                                #"?Schedule Flexibility (Fixed, Flexible)?" : "",
-                                "config_location" : dag.default_args["config_folder"] + "/" + config_file,
-                                #"Logs Location (filepath, email inbox, slack channel where success/failure is logged)" : "",
-                                "od_owner" : dag.owner,
-                                #"etl_description": config[package_name]["excerpt"],
-                                #"cot_owner" : config[package_name]["owner_division"],
-                                #"?Date Created?" : "",
-                                #"?Active/Inactive?" : "",
-                            })
+                            airflow_configs.append(
+                                {
+                                    "source_data": config[package_name][
+                                        "resources"][resource_name]["url"],
+                                    "target_package_name": package_name,
+                                    "target_resource_name": resource_name,
+                                    "engine": "Airflow - YAML Job",
+                                    "schedule": dag.schedule_interval,
+                                    "config_location": dag.default_args[
+                                        "config_folder"
+                                        ]
+                                    + "/"
+                                    + config_file,
+                                    "od_owner": dag.owner,
+                                }
+                            )
     print(airflow_configs)
     return {"output": airflow_configs}
-                        
-# Combine Configs w ckan resources
+
+
 def combine_configs(**kwargs):
+    """Puts configs together into single usable resource"""
     # init output
     output = []
 
     # get configs from other operators
-    nifi_configs = kwargs["ti"].xcom_pull(task_ids="get_nifi_configs")["output"]
-    airflow_configs = kwargs["ti"].xcom_pull(task_ids="get_airflow_configs")["output"]
+    nifi_configs = kwargs["ti"].xcom_pull(task_ids="get_nifi_configs")[
+        "output"
+        ]
+    airflow_configs = kwargs["ti"].xcom_pull(task_ids="get_airflow_configs")[
+        "output"
+        ]
+    fme_configs = kwargs["ti"].xcom_pull(task_ids="get_fme_configs")[
+        "output"
+        ]
 
     # make a master list of etl configs
-    configs = nifi_configs + airflow_configs
+    configs = airflow_configs + nifi_configs
```

Comment:
Should fme_configs also be added to configs?

Reply:
fair question, but no :) 
There are far fewer FME configs, and much less info I'm pulling from them right now, so I address adding FME config info here:

https://github.com/open-data-toronto/operations/blob/123df96272a71e6a26f64efa7b2e2ed3660d9a63/dags/sustainment/pull_etl_configs.py#L288

Reply:
ok, I see. So a boolean indicator is assigned to determine if that's a FME job. :)